app/views/application/_background.html.erb

```diff
@@ -0,0 +1,37 @@
+<%-
+  $background = "navbar-overlap"
+  $new_background = "navbar-overlap background"
+  $no_background = "navbar-overlap"
+  case (request.user_agent =~ /Mobile|webOS/).present? # check if user is using a mobile device
+    when true   # in case it is a mobile device
+      case controller_name
+        when ""
```

Comment:
Both conditional are equal. 'Case' can be left out.

Reply:
Done